### ISSUE TITLE
deb package reports incorrect Installed-Size

### ISSUE THREAD

**Issue by nicktime** [mi-g/vdhcoapp#23]
![net downloadhelper coapp properties as superuser _020](https://user-images.githubusercontent.com/217547/37872412-61c53774-3018-11e8-9dcd-f6b63567a837.png)
Shows up as 97.3 GB but should be around 90 MB.

**Comment by gianfp** [comment]
Hi,
just as a confirmation I got the same issue in elementaryOS (Ubuntu derivative).

![downloadhelper](https://user-images.githubusercontent.com/8994556/38519642-ce65bc08-3c40-11e8-9a45-c9078c6afb82.png)

**Comment by KINFOO** [comment]
Same from Debian stable command line:
```bash
$ sudo apt remove --purge net.downloadhelper.coapp
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages will be REMOVED:
  net.downloadhelper.coapp*
0 upgraded, 0 newly installed, 1 to remove and 3 not upgraded.
After this operation, 97.3 GB disk space will be freed.
Do you want to continue? [Y/n]
```

**Comment by mi-g** [comment]
This should have been fixed in [vdhcoapp 1.2.1](https://github.com/mi-g/vdhcoapp/releases/tag/v1.2.1)
